Avoid "Use of qw(...) as parentheses is deprecated" warnings

See http://search.cpan.org/~jesse/perl-5.14.0/pod/perl5135delta.pod#Use_of_qw%28...%29_as_parentheses
diff -ur IPTables-ChainMgr/lib/IPTables/ChainMgr.pm IPTables-ChainMgr-0.9/lib/IPTables/ChainMgr.pm
--- IPTables-ChainMgr/lib/IPTables/ChainMgr.pm	2009-02-12 05:20:16.000000000 +0100
+++ IPTables-ChainMgr-0.9/lib/IPTables/ChainMgr.pm	2012-01-06 15:20:15.055429198 +0100
@@ -155,7 +155,7 @@
         if ($extended_href) {
             $msg = "Table: $table, chain: $chain, $normalized_src -> " .
                 "$normalized_dst ";
-            for my $key qw(protocol s_port d_port mac_source) {
+            for my $key (qw(protocol s_port d_port mac_source)) {
                 $msg .= "$key $extended_href->{$key} "
                     if defined $extended_href->{$key};
             }
@@ -188,7 +188,7 @@
 
         $msg = "Table: $table, chain: $chain, added $normalized_src " .
             "-> $normalized_dst ";
-        for my $key qw(protocol s_port d_port mac_source) {
+        for my $key (qw(protocol s_port d_port mac_source)) {
             $msg .= "$key $extended_href->{$key} "
                 if defined $extended_href->{$key};
         }
@@ -244,7 +244,7 @@
         if ($extended_href) {
             $msg = "Table: $table, chain: $chain, $normalized_src -> " .
                 "$normalized_dst ";
-            for my $key qw(protocol s_port d_port mac_source) {
+            for my $key (qw(protocol s_port d_port mac_source)) {
                 $msg .= "$key $extended_href->{$key} "
                     if defined $extended_href->{$key};
             }
@@ -287,7 +287,7 @@
 
         $msg = "Table: $table, chain: $chain, added $normalized_src " .
             "-> $normalized_dst ";
-        for my $key qw(protocol s_port d_port mac_source) {
+        for my $key (qw(protocol s_port d_port mac_source)) {
             $msg .= "$key $extended_href->{$key} "
                 if defined $extended_href->{$key};
         }
@@ -344,7 +344,7 @@
 
     my $extended_msg = '';
     if ($extended_href) {
-        for my $key qw(protocol s_port d_port mac_source) {
+        for my $key (qw(protocol s_port d_port mac_source)) {
             $extended_msg .= "$key: $extended_href->{$key} "
                 if defined $extended_href->{$key};
         }
@@ -403,13 +403,13 @@
                 and $rule_href->{'dst'} eq $dst) {
             if ($extended_href) {
                 my $found = 1;
-                for my $key qw(
+                for my $key (qw(
                     protocol
                     s_port
                     d_port
                     to_ip
                     to_port
-                ) {
+                )) {
                     if (defined $extended_href->{$key}) {
                         unless ($extended_href->{$key}
                                 eq $rule_href->{$key}) {
